src/components/Registration/BusinessTypeConfirm.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div id="business-type">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label class="d-block title-label">Business Type</label>
+        <v-chip
+          v-if="hasBusinessTypeChecked"
+          id="checked-chip"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          Checked
+        </v-chip>
+      </v-col>
+
+      <v-col cols="12" sm="9">
+        <p class="mb-0">General Partnership</p>
+        <div id="business-type-div">
+          <v-checkbox
+            class="business-type-check mt-0"
+            hide-details
+            v-model="checked"
+          />
+          <p class="mb-0" id="business-type-text">
+            I acknowledge that a General Partnership cannot be changed into a
+            Sole Proprietorship (including DBA). If this is necessary, a new
+            Name Request Number and Statement of Registration (along with
+            associated fees) will be required.
+          </p>
+        </div>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+
+@Component
+export default class BusinessTypeConfirm extends Vue {
+  /** Whether to display Change features. */
+  @Prop({ default: false })
+  readonly hasBusinessTypeChecked!: boolean;
+
+  // Local variables
+  protected checked = false;
+
+  /** Watches for changes to the checkbox value. */
+  @Watch('checked')
+  private onValueChanged (): void {
+    // update business type + validate
+    this.emitBusinessType()
+    this.emitValid()
+  }
+
+  /** Emits updated business type event. */
+  @Emit('update:BusinessTypeConfirm')
```

Comment:
This needs to be camelCase too.

I'm a bit surprised that the parent event spec (`@update:businessTypeConfirm=...`) works. I know it auto-converts from camelCase to kebab-case but this is new to me 🤷‍♂️